### DISCUSSION TITLE
Error too many levels of symbolic links

### DISCUSSION THREAD

**Discussion by DaPhreak1980** [fabianishere/udm-iptv#214] [Q&A]
Hi Fabian,
First of all thanks a lot for all your work here. I have an issue with the install.sh and I don't know how to solve:

root@DreamMachinePro:~# sh -c "$(curl https://raw.githubusercontent.com/fabianishere/udm-iptv/master/install.sh -sSf)"
Downloading packages...
Installing packages...
Hit:1 http://security.debian.org stretch/updates InRelease
Ign:2 http://deb.debian.org/debian stretch InRelease
Hit:3 http://deb.debian.org/debian stretch-updates InRelease
Hit:4 http://deb.debian.org/debian stretch-backports InRelease
Hit:5 http://deb.debian.org/debian stretch Release
Hit:7 https://apt-release-candidate.artifacts.ui.com stretch InRelease
Hit:8 https://apt.artifacts.ui.com stretch InRelease
Hit:9 https://apt-beta.artifacts.ui.com stretch InRelease
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
dialog is already the newest version (1.3-20160828-2).
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up udm-iptv (2.1.0) ...
dpkg: warning: udm-iptv: unable to stat config file '/etc/udm-iptv.conf'
 (= '/mnt/persistent/udm-iptv.conf'): Too many levels of symbolic links
dpkg: warning: udm-iptv: unable to open /mnt/persistent/udm-iptv.conf for hash: Too many levels of symbolic links
dpkg: error processing package udm-iptv (--configure):
 unable to stat new distributed conffile '/mnt/persistent/udm-iptv.conf.dpkg-new': Too many levels of symbolic links
Errors were encountered while processing:
 udm-iptv
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by fabianishere** [comment]
Try

`rm /etc/udm-iptv.conf`

**Reply by DaPhreak1980** [reply]
thanks worked!
